linkml/generators/excelgen.py

```diff
@@ -37,49 +37,41 @@ def create_workbook(self, workbook_name: str) -> Workbook:
         :return: An openpyxl Workbook object representing the newly created workbook.
         """
         workbook = Workbook()
-        workbook.title = workbook_name
+        workbook.save(workbook_name)
         return workbook
 
-    def get_workbook_name(self, workbook: Workbook) -> str:
+    def create_workbook_and_worksheets(self, output_path: str, classes: List[str]) -> None:
```

Comment:
You have type `str` for a path her. If you use pathlib (of which I am not sure) it would be better to use `path` type.